tracer/operation/hassuicided.go

```diff
@@ -59,7 +59,8 @@ func (op *HasSuicided) Write(f io.Writer) error {
 func (op *HasSuicided) Execute(db state.StateDB, ctx *context.Replay) time.Duration {
 	contract := ctx.DecodeContract(op.Contract)
 	start := time.Now()
-	db.HasSuicided(contract)
+	//TODO change function name
```

Comment:
nit: todo?

Reply:
An upgrade to tracer and profiler will be added in the next PR.